src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,18 +11,18 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
 
     /** The list of persons that was produced by the command */
     private final List<? extends ReadOnlyPerson> relevantPersons;
 
     public CommandResult(String feedbackToUser) {
-        this.feedbackToUser = feedbackToUser;
+        this.feedbackToUser = getFeedbackToUser();
```

Comment:
This is not necessary. Method of the same class can access members of all types of visibility, i.e. private, protected, public.